src/engraving/rw/read460/tread.cpp

```diff
@@ -3764,11 +3764,8 @@ void TRead::read(StaffType* t, XmlReader& e, ReadContext& ctx)
         } else if (tag == "clef") {
             t->setGenClef(e.readInt());
         } else if (tag == "stemless") {
-            bool val = e.readInt() != 0;
             t->setStemless(e.readBool());
-            t->setShowBackTied(!val);        // for compatibility with 2.0.2 scores where this prop
```

Comment:
I originally made this same change, but then reverted it because of what `TWrite::write(const StaffType*` does:
```c++
        // only output "showBackTied" if different from !"stemless"
        // to match the behaviour in 2.0.2 scores (or older)
        if (item->showBackTied() != !item->stemless()) {
            xml.tag("showBackTied",  item->showBackTied());
        }
```

But I made a mistake while reverting my change, because I forgot to change `e.readBool()` back into `val`.

Reply:
Ah, thanks!